.github/validate-rapps.py

```diff
@@ -125,17 +131,19 @@ def _parse_section(self, reporter, stripped):
 
         if section_name != b'Section':
             help = 'should always be "Section"'
-            reporter.add(self, self._text.index(section_name) + 1, f'Invalid section name: "{section_name}", {help}')
+            reporter.add(self, self._text.index(section_name) + 1,
+                         'Invalid section name: "{sec}", {msg}'.format(sec = section_name, msg = help))
         elif not locale:
             self.main_section = True
 
         if locale:
             if len(locale) not in (2, 4) or not all(c in HEXDIGITS for c in locale):
-                reporter.add(self, self._text.index(locale) + 1, f'Invalid locale{extra_locale}: "{locale}"')
+                reporter.add(self, self._text.index(locale) + 1,
+                             'Invalid locale{extra}: "{loc}"'.format(extra = extra_locale, loc = locale))
 
         if arch:
             if arch not in ALL_ARCH:
-                reporter.add(self, self._text.index(arch) + 1, f'Unknown architecture: "{arch}"')
+                reporter.add(self, self._text.index(arch) + 1, 'Unknown architecture: "%s"' % arch)
```

Comment:
Why are you using a different format method here?

Reply:
Because the syntax is shorter and there is no ambiguity about which argument goes where.